war/src/main/less/modules/section.less

```diff
@@ -103,14 +103,12 @@
       display: flex;
       align-items: center;
       justify-content: center;
-      background: #ff0b27;
       border-radius: 100px;
       font-size: 0.7rem;
       min-height: 20px;
       min-width: 20px;
       padding: 0 0.4rem;
       box-shadow: 0 1px 1px rgba(black, 0.1);
-      color: white;
```

Comment:
Does this only affect Manage Jenkins badges? `.jenkins-section` looks very generic.